cmd/api-endpoint/expected_output_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+const SUBMISSION_1697406272 = `{
```

Comment:
!
This is duplicated in cmd/peek-submission.

(Yellow flag, this is clearly duplicate code and you wrote both.)